x-pack/platform/plugins/private/translations/translations/de-DE.json

```diff
@@ -3896,39 +3896,6 @@
     "grouping.nullGroup.title": "Das ausgewählte Gruppierungsfeld, {selectedGroup}, hat keinen Wert für diese Gruppe von {unit}.",
     "grouping.selector.grouping.label.add": "Hinzufügen",
     "grouping.total.unit": "{groupCount} {totalCount, plural, =1 {group} other {groups}}",
-    "guidedOnboarding.dropdownPanel.backToGuidesLink": "Zurück zu den Leitfäden",
```

Comment:
I'd expect this to be removed in the final PR. I'm not sure if this is very important, but I wanted to mention it just in case.